cas/incident-report.md

```diff
@@ -1,169 +1,384 @@
+Either:
+1) disclose a Preliminary or Full Incident Report; or
+2) respond to a Preliminary Incident Report previously created for the incident by a third party reporter.
 
-- Policy requires the revocation of one or more certificates by a certain deadline, such as those in BR section 4.9, but that deadline will not be or has not been met by the CA Owner.
-- In the process of researching one incident, another incident with a distinct root cause and/or remediation is discovered.
-- After an incident is marked resolved in Bugzilla, the incident reoccurs.
+And:
+- accurately disclose the impact of the incident (e.g., a corpus of mis-issued certificates).
+- describe whether the incident should be considered contained (e.g., because certificate issuance was stopped) or ongoing.
 
-The incident report may well repeat things previously said in discussions or Bugzilla comments. This is entirely expected. The report should be a summary of previous findings. The existence of data in discussions or Bugzilla comments does not excuse a CA Owner from the task of compiling a proper incident report.
+If the described impact of the incident is later found to be inaccurate, the CA Owner MUST clearly communicate a correction, identifying when each following change was detected, the circumstances that led to the inaccuracy, and how this will be avoided in the future.
 
-Open incident reports should be updated:
+While Full Incident Reports SHOULD be posted as soon as possible, they MUST be posted within 14 days of the incident’s initial disclosure.
 
-- on or before the "Next update" date in the "Whiteboard" field of the bug;
-- weekly, if a "Next update" date is not recorded; or
+### What is considered an incident?
+
+Minimally, a failure to meet the commitments described in any of the following policies is considered an incident:
+- A CA Owner's own policies (e.g., CP, CPS, or combined CP/CPS);
+- Applicable requirements promulgated by the CA/Browser Forum;
+- The CCADB Policy; or
+- Any applicable Root Store Operator policy.
+
+Root Store Operator policies MAY further describe what those individual programs consider incidents, and/or additional incident reporting expectations.
+
+### When should Incident Reports be updated?
+
+CA Owners SHOULD respond promptly to comments and questions, and in no circumstances MUST a request or question linger without a response for more than 7 days, even if the response is only to acknowledge the request or question and provide a later date when a response will be delivered.
+
+Open Incident Reports MUST be updated:
+- on or before the "Next update" date in the "Whiteboard" field of the bug (note: CA Owners MAY request the "Next update" Whiteboard field be set by a Root Store Operator to align with a specific date related to an open Action Item.);
+- weekly, if a "Next update" date is not recorded;
```

Comment:
Given the more prescriptive requirements being added elsewhere, I wonder if "weekly" is sufficiently descriptive of this expectation. For example, can someone post an update on a Monday and then on Friday of the following week? I'm not a fan of making this strictly 'within 7 days' either. Perhaps SHOULD weekly and MUST within 14 days would be a good balance here.

Reply:
Or just have the MUST be 7 or 10 days.

Reply:
#187 updates this to...

"_In the case of Incident Reports with a Whiteboard field of "revocation-delay", reports SHOULD be updated every 3 days and MUST be updated no less frequently than every 7 days to describe a summary of..._"